doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -25,7 +62,7 @@ $DocOutHtmlDir = "${DocOutDir}/_site"
 $MDocTool = "${BinDirectory}/mdoc/mdoc.exe"
 $DocFxTool = "${BinDirectory}/docfx/docfx.exe"
 
-if ($LibType -eq 'Management') {
+if ($LibType -eq 'management') {
```

Comment:
Tip: Just FYI, comparisons in PowerShell are, by default, case-insensitive. You can use operators like `-ceq` for case-sensitive, or `-ieq` for case-insensitive (which I never understood why they have this, since `-eq` is case-insensitive).